ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -383,7 +394,9 @@ export default function TransactionList({
             ) : (
               <Box className="transaction-list__empty">
                 <Box className="transaction-list__empty-text">
-                  {t('noTransactions')}
+                  {isChainIdMismatch
+                    ? noTransactionsMessage
+                    : t('noTransactions')}
```

Comment:
don't have context here, but do we only want to show this message when there are 0 transactions?

Reply:
Actually.. it will have 0 transactions since it is the incorrectly selected chain. 👍🏾